sync2/fptree/export_test.go

```diff
@@ -7,6 +7,16 @@ import (
 
 var ErrEasySplitFailed = errEasySplitFailed
 
+func (ft *FPTree) FingerprintIntervalInternal(
```

Comment:
NIT:

```suggestion
func (ft *FPTree) FingerprintInternal(
```

Reply:
There's already `FingerprintInterval` method which doesn't accept `needNext` argument, as `needNext` is only used internally by `FPTree`, so I named this test-only method `FingerprintIntervalInternal`.

Reply:
If its test only why is it exported? How about naming it `testFingerprintInterval`?

Reply:
It's exported b/c of blackbox testing, so that `fptree_test` package can access it

Reply:
Similar to e.g. https://go.dev/src/runtime/export_test.go https://go.dev/src/net/http/export_test.go

Reply:
Fixed (along with typos)